backend/coins/btc/handlers/handlers.go

```diff
@@ -410,7 +409,11 @@ func (input *sendTxInput) UnmarshalJSON(jsonBytes []byte) error {
 }
 
 func (handlers *Handlers) postAccountSendTx(r *http.Request) (interface{}, error) {
-	err := handlers.account.SendTx()
+	var finalNote string
+	if err := json.NewDecoder(r.Body).Decode(&finalNote); err != nil {
+		handlers.log.WithError(err).Error("Failed to unmarshal transaction note")
```

Comment:
note: I decided to ignore the error on purpose here and only log it. it is better if users contact us with a _bug that their notes do not get applied_ rather then _I can't send any coins anymore_

this should not happen anyways, and if it does proceeding with the zero string value is like the user never entered anything into the note field.

Reply:
*what I mean is I don't return an error here and continue on purpose

Reply:
please add this as a comment, otherwise the next person to look at this will think it's a bug and return an error :smile: